scopesim/effects/ter_curves.py

```diff
@@ -201,7 +204,12 @@ def plot(self, which="x", wavelength=None, *, axes=None, **kwargs):
         if wavelength is None:
             wunit = params["wave_unit"]
             # TODO: shouldn't need both, make sure they're equal
-            assert wunit == wave_unit
+            if wunit != wave_unit:
+                logger.warning(f"wavelength units in the meta dict of "
+                             f"{self.meta.get('name')} are inconsistent: \n"
+                             f"- wavelength_unit : {wave_unit} \n"
+                             f"- wave_unit : {wunit}")
```

Comment:
```suggestion
                logger.warning("wavelength units in the meta dict of "
                             "%s are inconsistent:\n"
                             "- wavelength_unit : %s\n"
                             "- wave_unit : %s",
                             {self.meta.get("name")},
                             wave_unit, wunit)
```
No f-strings in logging...